python/pyfury/_fury.py

```diff
@@ -82,11 +82,15 @@
 # Key is null, value type is declared type, and ref tracking for value is disabled.
 NULL_KEY_VALUE_DECL_TYPE = KEY_HAS_NULL | VALUE_DECL_TYPE
 # Key is null, value type is declared type, and ref tracking for value is enabled.
-NULL_KEY_VALUE_DECL_TYPE_TRACKING_REF = KEY_HAS_NULL | VALUE_DECL_TYPE | TRACKING_VALUE_REF
+NULL_KEY_VALUE_DECL_TYPE_TRACKING_REF = (
```

Comment:
could we reserve those parameters in _serializer.py, it's used by map serializer only